docs-kits/kits/PURIS Kit/page_adoption-view.md

```diff
@@ -1,232 +1,187 @@
 
-- **LEs**: Enhanced tracking and coordination of incoming and outgoing shipments, enabling better production planning and inventory management.
-- **SMEs**: Improved visibility into delivery timings and quantities helps in adjusting operations, reducing storage costs, and enhancing customer satisfaction.
-- **Solution Providers**: Ability to offer more accurate and timely logistics services, enhancing value to clients through better supply chain transparency and efficiency.
+| Semantic Model                                                                                                                         | Version | Standard name | Description                                                                                                                                                                                                                                                                                           |
+| -------------------------------------------------------------------------------------------------------------------------------------- | :-----: | :-----------: | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Item Stock](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.item_stock)                                 | v.2.0.0 |    CX-0122    | Item Stock includes materials in the outbound warehouse or inbound at the customer's site. Standardizing its definition and using digital twins for data sharing allows quick inventory updates across the supply chain, improving coordination and addressing supply shortages.                      |
+| [Short-Term Material Demand](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.short_term_material_demand) | v.2.0.0 |    CX-0120    | This exchange specifies the material a customer's production line needs from a supplier in the next four weeks. It adds context to regular orders, highlighting essential demands and extra requests for safety stocks. This detail helps suppliers plan and address potential shortages proactively. |
+| [Planned Production Output](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.planned_production_output)   | v.2.0.0 |    CX-0121    | This exchange shows the agreed but unmanufactured material quantity a supplier plans to produce for a customer. Standardizing this via digital platforms ensures clear communication of future production volumes, aligning supply chain expectations and improving responsiveness to market needs.   |
```

Comment:
Here the aspect model and the standard are both v2.0.0

Suggestion: update the link to the currently standardized semantic model: https://github.com/eclipse-tractusx/sldt-semantic-models/tree/4d239fc5709f71f39c3cf13581b5bcf960905157/io.catenax.planned_production_output/2.0.0

Otherwise it is easy to confuse the version of the standard, with the version of the aspect model that is actually standardized.

Reply:
Link has been adapted